SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -323,9 +327,25 @@ private void addCredentials(HttpRequest.Builder requestBuilder) {
     return Optional.absent();
   }
 
+  private <T> Optional<T> put(String uri, String type, Optional<?> body, Optional<Class<T>> clazz) {
```

Comment:
it looks like this method is unused, and is causing findbugs to barf. could you remove this method to make the build pass?